### PR TITLE
feat: add a line

### PR DESCRIPTION
**Pull request by dawhum**

### CODE REVIEW

**Comment by dawhum** [comment]
@addrum I think you need to add a comment to the new code. I can't even find the dismiss button as it stands.

File: test2 [hunk 1]
```diff
@@ -1,2 +1,4 @@
 test2
 test3
+
```
Comment (addrum) [inline test2]: I don't like this new line

Comment (addrum) [reply test2]: Pretty sure I added it as a review comment.

Comment (dawhum) [reply test2]: Argh, I think I know what's up. You're not a member of the repo,  so your reviews are only suggestions. So confusing. Sorry, mate.

Comment (addrum) [reply test2]: hopefully this isn't a suggestion now...

**Comment by dawhum** [comment]
That worked. Cheers!

**Comment by dawhum** [comment]
qwe

File: test2 (hunk 1 above)
Comment (dawhum) [reply test2]: @addrum Please could you approve this for me?

**Pull request merged**